label_studio/ml_model_providers/models.py

```diff
@@ -42,3 +43,20 @@ class ModelProviderConnectionScopes(models.TextChoices):
 
     def has_permission(self, user):
         return user.active_organization == self.organization
+
+    def validate_api_key(self):
```

Comment:
please add a type hint here. I usually prefer `validate_...` functions to raise or return None; you could instead call it `api_key_is_valid` and have return type `bool`